zombienet_tests/0004-runtime_upgrade.toml

```diff
@@ -0,0 +1,35 @@
+[relaychain]
+default_image = "docker.io/paritypr/polkadot-debug:master"
+default_command = "polkadot"
+default_args = [ "-lparachain=debug" ]
+
+chain = "rococo-local"
+
+  [[relaychain.nodes]]
+  name = "alice"
+  validator = true
+
+  [[relaychain.nodes]]
+  name = "bob"
+  image = "docker.io/paritypr/polkadot-debug:5236-0.9.18-c55660e9-be16bd72"
```

Comment:
Why do we need this image here?

Reply:
Hi @bkchr, no this image should be set as `env var` by the ci job. The `ci` isn't ready yet, I will update this in all the files.

Thanks!